pkgs/by-name/ho/hours/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "hours";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "dhth";
+    repo = "hours";
+    tag = "v${finalAttrs.version}";
```

Comment:
I believe this is an anti-pattern, see https://github.com/NixOS/nixpkgs/issues/310373.  It's better to just use `rec` and `version`.